### DISCUSSION TITLE
Can I use the roblox website to lunch game with sober

### DISCUSSION THREAD

**Discussion by Balder559** [vinegarhq/sober#535] [Q&A]
I deeply prefer using the Roblox website as i hate the homepage layout in the app but using ublock on website i can you it too get close to the O.G layout so can I use the roblox website to lunch game with sober

**Comment by ultradev-coder** [comment]
you can use the website to launch roblox games with sober and it works perfectly fine for me.

**Reply by Balder559** [reply]
how do i do so im on firefox

**Reply by ultradev-coder** [reply]
make sure you're logged in on the sober app, go to the roblox website, select your game and click play

**Reply by Balder559** [reply]
Nmv got it work
